src/main/java/uk/gov/companieshouse/ordernotification/emailsendmodel/OrderNotificationEmailDataConverter.java

```diff
@@ -104,4 +103,21 @@ private void mapDeliveryTimescale(DeliveryTimescaleApi deliveryTimescaleApi) {
             emailData.hasExpressDelivery(true);
         }
```

Comment:
should we handle the else case by throwing a MappingException or NonRetryableFailureException?

Reply:
Jackson will already have thrown a `JsonMappingException` by this point if an invalid enum value has been specified.